packages/playground/src/components/Playground.tsx

```diff
@@ -0,0 +1,410 @@
+import { useCallback, useState, useRef, useEffect, type ComponentType, type FormEvent } from 'react';
+import 'react-app-polyfill/ie11';
+import isEqualWith from 'lodash/isEqualWith';
+import Form, { withTheme, type IChangeEvent, type FormProps } from '@rjsf/core';
+import type {
+  ErrorSchema,
+  TemplatesType,
+  ArrayFieldTemplateProps,
+  ObjectFieldTemplateProps,
+  RJSFSchema,
+  UiSchema,
+} from '@rjsf/utils';
+import localValidator from '@rjsf/validator-ajv8';
+
+import { samples } from '../samples';
+import DemoFrame from './DemoFrame';
+import ErrorBoundary from './ErrorBoundary';
+import CopyLink from './CopyLink';
+import GeoPosition from './GeoPosition';
+import ThemeSelector from './ThemeSelector';
+import Selector from './Selector';
+import ValidatorSelector from './ValidatorSelector';
+import SubthemeSelector from './SubthemeSelector';
+import RawValidatorTest from './RawValidatorTest';
+import Editor from './Editor';
+
+const log = (type: string) => console.log.bind(console, type);
+const toJson = (val: unknown) => JSON.stringify(val, null, 2);
+
+const liveSettingsSchema = {
```

Comment:
If you add this type you may not need the cast to `any` below
```suggestion
const liveSettingsSchema: RJSFSchema = {
```